extensions/amp-story/1.0/test/test-amp-story-bookend.js

```diff
@@ -365,6 +367,99 @@ describes.realWin('amp-story-bookend', {amp: true}, env => {
     });
   });
 
+  it('should fire analytics event when clicking on a link', () => {
+    const userJson = {
+      'bookendVersion': 'v1.0',
+      'shareProviders': [
+        'email',
+        {'provider': 'facebook', 'app_id': '254325784911610'},
+        'whatsapp',
+      ],
+      'components': [
+        {
+          'type': 'cta-link',
+          'links': [
+            {
+              'text': 'buttonA',
+              'url': 'google.com',
+              'amphtml': true,
+            },
+          ],
+        },
+      ],
+    };
+
+    sandbox.stub(requestService, 'loadBookendConfig').resolves(userJson);
+    const analyticsSpy = sandbox.spy(bookend.analyticsService_, 'triggerEvent');
+    const variableSpy = sandbox.spy(
```

Comment:
Can we let the variable service do its magic and assert on values from `variableService.get(...)`?